prow/config.yaml

```diff
@@ -317,11 +317,9 @@ branch-protection:
               - "E2E (purge-metrics)"
               - "E2E (watcher-enqueue)"
               - "Run 'make test'"
-              - "kyma (unstable) alpha deploy -k config/control-plane"
               - "kyma (unstable) alpha deploy -k config/default"
```

Comment:
both deploy test can be removed

Reply:
above statement is wrong, we need to keep both tests, but remove "Run Smoke Tests " step, only test execution of  kyma alpha deploy